### ISSUE TITLE
Context in EU Standards

### ISSUE THREAD

**Issue by Reccetech** [TalaoDAO/hedera-vc-api#3]
Hello. How does hedera-vc-api fit in with the evolving EU identity standards. My quick perspective is that most of the EBSI "standards" focus on the OpenID4VC and OpenID4Issuance interactions. So hedera-vc-api can be used to create the VCs that can then be put into an identity token, create the VP to respond to a proof request, and be used to verify the received VP from a token. 
That being said there are many profiles for a VC. Looking at the jwt_vc_json samples like here I see some differences from the VC data model in hedera-vc-api. So is there an "EU" VC data model this is based on - or another VC profile spec I can reference?
Thanks!

**Comment by ThierryThevenet** [comment]
Yes, this implementation is based on https://github.com/w3c-ccg/vc-api which is still focused on ldp_vc. To be as close as possible we have decided to use  the same data model. It is also the data model used by The Guardian.

But in practice we are using the hedera-vc-api for the EON/Greencypher project which uses a jwt_vc_json-ld data model. We just use the server to generate a DID and manage the revocation list. The  signature/verification of our VCs (jwt_vc_json-ld)  is done in the application code with a JOSE lib. The protocol is OIDC4VCI + OIDC4VP .
So yes, 100% agree, the next step would be to integrate the jwt format inside the hedera vc api server. I would also suggest to move directly to the sd-jwt data model ( Baseprofile proposed in the OWF due diligence wg or GAIN-POC wg). That would cover at minimum all the EU future projects.
EBSI is still using a standard jsw_vc_json) but next release will be sd-jwt.

**Comment by ThierryThevenet** [comment]
sd-jwt https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-01.html will be supported by our wallets in Q1 next year.

Update of the OIDC4VC current profiles : https://docs.google.com/spreadsheets/d/1s6REK5eNAb3GSElID0J02_TtbuI2Exd9z-CLdLx0emk/edit#gid=0 ...**.with sd-jwt almost every where now.**
